tests/BasicSearchTests.FunctionalTests.Core/V3SearchFunctionalTests.cs

```diff
@@ -37,5 +37,23 @@ public class V3SearchFunctionalTests : BaseFunctionalTests
             Assert.True(result.TotalHits.HasValue && result.TotalHits.Value > 0, "No results found, should find atleast some results for empty string query.");
             Assert.NotNull(result.Data);
         }
+
+        [Fact]
+        public async Task ResultForJsonNetHasDownloads()
+        {
+            // Act
+            var response = await Client.GetAsync(new V3SearchBuilder() { Query = "JSON.Net" }.RequestUri);
```

Comment:
This won't necessarily check the download count for JSON.net, rather the close match of the package. In INT and DEV environment we do not have any JSON.Net package. Also, in local testing this test would fail because we won't have any data necessarily for this query. 

Honestly, I see very low value in this test. If we do want to check some data exists on a query, I would rather do an empty string query and test the download count, which again is possibly useless when running this locally(which arguably is not the idea of functional tests).